Operational-Procedures/Main-Telescope/ComCam-Master-Bias-Procedure/comcam-master-bias-procedure.rst

```diff
@@ -0,0 +1,110 @@
+.. |author| replace:: *Andrés A. Plazas Malagón*
+.. If there are no contributors, write "none" between the asterisks. Do not remove the substitution.
+.. |contributors| replace:: *none*
+
+.. _ComCam-Master-Bias-Procedure:
+
+###########################
+ComCam Master Bias Generation Procedure
+###########################
+
+.. _ComCam-Master-Bias-Procedure-Overview:
+
+Overview
+========
+
+This procedure describes how to call scripts to produce a master bias for LSSTComCam from the LSST Operations and Visualization Enviroment (LOVE) at the Summit :ref:`operational environment <Observing-Interface-Operational-Environments>`. 
+
+The script will: 
+
+- command LSSTComCam to take a number of bias frames,
+- call the Rubin Science Pipelines bias generation pipetask via the OCS-Controlled Pipeline System (OCPS),
+- verify the resulting bias (see the package `cp_verify`_ and `DMTN-101`_)
+- certify the resulting bias with a given range of validity dates.
+  
+For more information about calibrations production (including verification and certification), please consult the `Constructing Calibrations documentation`_.
+
+.. _cp_verify: https://github.com/lsst/cp_verify
+.. _DMTN-101: https://dmtn-101.lsst.io/
+.. _Constructing Calibrations documentation: https://lsst.ncsa.illinois.edu/~czw/pipelines_lsst_io/_build/html/modules/lsst.cp.pipe/constructing-calibrations.html 
+
+.. _ComCam-Master-Bias-Procedure-Prerequisites:
+
+Prerequisites
+=============
+
+- You should be logged into the LSST Operations and Visualization Enviroment (LOVE) at the Summit :ref:`operational environment <Observing-Interface-Operational-Environments>`.
+- The script assumes  that the LSSTComcam and OCPS are both ``ENABLED``, and that latter has been ``ENABLED`` with the configuration of ``LSSTComCam``.
+- A `butler`_ is needed to run the pipetask bias command.
```

Comment:
I am not really sure I understand this point here. Does the Script needs access to butler in order to run or is it the OCPS?

In any case, I think this is a general environment consideration that is probably out of the scope of this procedure. 

I think you could remove it.

Reply:
I was referring to the fact that the pipetask needs a repo such as ``/repo/LSSTComCam`` and that the stack should be loaded. But I think you are correct;  I'll remove it per your consideration.